db-ops/create-db-user.sh

```diff
@@ -53,6 +53,7 @@ fi
 JOB_ID=$(LC_ALL=C tr -dc 'a-z0-9' < /dev/urandom | head -c 8)
 eval "echo \"$(cat ./db-ops/job-create-db-${DATABASE_TYPE}.yml.tpl)\"" > ./k8s-job-create-db.yml
 [[ -z "${CREATE_SECRET}" ]] || eval "echo \"$(cat ./db-ops/${CREATE_SECRET})\"" >> ./k8s-job-create-db.yml
+[[ -z "${CREATE_DB_POD}" ]] || eval "echo \"$(cat ./db-ops/deployment-db-test.yml.tpl)\"" >> ./k8s-job-create-db.yml
```

Comment:
Why was this pulled out into another file?

Reply:
considered not to run db-pod for developers, so made it separate.  but now refreshed and removed.